src/Services/ConvertToQtiService.php

```diff
@@ -194,12 +199,15 @@ private function convertAssessmentItem($json)
         $content = $json['content'];
         $tags = $json['tags'];
         $itemReference = $json['reference'];
+
         foreach ($json['questions'] as $question) :
             $question['content'] = $content;
             $question['itemreference'] = $itemReference;
             if (in_array($question['data']['type'], LearnosityExportConstant::$supportedQuestionTypes)) {
                 $result = Converter::convertLearnosityToQtiItem($question);
                 $result[0] = str_replace('/vendor/learnosity/itembank/', '', $result[0]);
+                $result[0] = str_replace('xmlns:default="http://www.w3.org/1998/Math/MathML"', '', $result[0]);
+                $result[0] = str_replace('default:', '', $result[0]);
```

Comment:
After we review again, we should probably specify tags that we targets in case we risks the recession.
Would you please specify the tags?
e.g.
```
$result[0] = str_replace('default:mi', '', $result[0]);
```

Reply:
By any chance, could you please fix the annotation at line 188?
from:
```
* @param  string $jsonString
```
to:
```
* @param  array $json
```

Reply:
Second part is done .
But we can not put whole tag like default:mi as search string in str_replace since after default : any math tag like mi, mo mfrac etc can come .

Reply:
Hi @rakeshqdsa ,
Just a thought, the conversion library add prefix `default:` to the related math tags, and we replace them later.
Is it possible that we don't add prefix or remove it in the first place during conversion?

Reply:
We are not adding any prefix, conversion library automatically adding this prefix .